go.mod

```diff
@@ -93,11 +93,11 @@ require (
 	go.opentelemetry.io/otel/sdk v1.27.0
 	go.uber.org/automaxprocs v1.5.3
 	go.uber.org/goleak v1.3.0
-	golang.org/x/crypto v0.24.0
+	golang.org/x/crypto v0.31.0
```

Comment:
I would refrain from updating any core libraries in this fix. Maybe that can be done in a separate PR. Can you revert them if possible?

Reply:
Okay my bad. It seems like they were introduced because of a change in base branch.